similarity-search/src/main/java/org/phenotips/similarity/internal/SolrSimilarPatientsFinder.java

```diff
@@ -73,9 +73,6 @@
     /** The number of records to fully score. */
     private static final int SEED_QUERY_SIZE = 50;
```

Comment:
Hm, small number of seeds was introduced to help performance when there was only the in-patient-record instantly-computed similar cases. This needed a fast computation time, since users don't want to wait 5 minutes to see similar patients. But the way we compute and display matches has changed since then, so we need to re-evaluate this.

For all matches table, we don't care if the computation takes longer, since it's something we already expect to take a lot of time and we're fine with it taking a whole night.

@allasm For the in-patient-record similar cases, do we still compute matches each time the page is loaded?

We could improve the pool of seeds in three ways:
1. Increase the number of seeds
2. Limit the results by a Solr score threshold instead of by absolute count
3. Don't seed at all, but instead compute everybody-vs-everybody scores, which will make it impossible to miss similar cases (we have an open issue about missed similar cases because of slightly different phenotypes, and will also make exome-only matching possible); this is the costliest, but if it doesn't slow down things too much, now that we have incremental searches, may be the best option